src/legacy/server/saved_objects/migrations/core/elastic_index.test.ts

```diff
@@ -430,32 +427,28 @@ describe('ElasticIndex', () => {
       ];
 
       callCluster
-        .onCall(0)
-        .returns(
+        .mockReturnValueOnce(
```

Comment:
nit: There's also `mockResolvedValueOnce` that removes the need to add `Promise.resolve`

Reply:
replaced mockReturnValue's with mockResolvedValue's thanks!